server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequest.java

```diff
@@ -80,7 +85,11 @@ public GetSnapshotsRequest(StreamInput in) throws IOException {
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
-        out.writeString(repository);
+        if (out.getVersion().onOrAfter(Version.V_7_1_0)) {
```

Comment:
This should fail BwC tests shouldn't it? I think we gotta first set it to `V8_0_0` and then back port and adjust the constant?

Reply:
@original-brownbear thanks, I've fixed it in https://github.com/elastic/elasticsearch/pull/41799/commits/569131569fb254a58bcf0bc79a29c9fcddca6cf0.
The sequence of BwC story would be the following:
1) Merge this PR.
2) Create separate PR to disable snapshot related BwC tests in master and change version to 7.2.0.
3) Backport the change to 7.2.0.
4) Enable tests.